### ISSUE TITLE
BACKEND: running pytest coverage test crash currently running backend server

### ISSUE THREAD

**Issue by sokampdx** [codeforpdx/dwellingly-app#252]
This is happening at latest development branch (f455a67). 
Follow README to setup the dwellinglybackend environment.

Start the backend server or reload the backend server:
`pipenv run flask run --reload`

Run pytest coverage:
`pipenv run pytest --cov .`

The following things happend:
-  data.db file would updated 
-  the backend server would crash with error similar to this
`...
  File "/Users/kso/.local/share/virtualenvs/dwellinglybackend-n8ZLeKs6/lib/python3.8/site-packages/sqlalchemy/engine/default.py", line 588, in do_execute
    cursor.execute(statement, parameters)
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such table: revoked_tokens
[SQL: SELECT revoked_tokens.id AS revoked_tokens_id, revoked_tokens.jti AS revoked_tokens_jti 
FROM revoked_tokens 
WHERE revoked_tokens.jti = ?
 LIMIT ? OFFSET ?]
[parameters: ('ee0dee3b-db48-417e-8e5a-de186f770b6c', 1, 0)]
(Background on this error at: http://sqlalche.me/e/e3q8)
`

The behavior we should get is
- data.db should remove from git repo and clear and re-seed when test is done.
- the server should not crash after running test (this might be fix by fixing the first item)

**Comment by asasha** [comment]
Sorry for picking this up and putting back down, but I can't recreate this. I am able to run the flask app & then run the tests without any crashing. My steps:
```
% pipenv run flask run --reload
* new process *
% pipenv run pytest --cov .

(tests run)

================== 68 passed in 11.62s =====================

```

Is there something I am missing?

**Comment by NickSchimek** [comment]
@sokampdx I also cannot reproduce this error.

Although @sokampdx and @asasha I think I know what is going on here. This may have to do with the fact that we only have one environment setup, and that is production by default for Flask. Which can cause these types of errors when certain events are played out. I have an issue opened, that I am going to start working on, which will take care of this. Closing this issue as it cannot be reproduced and the root cause of this failure will most likely be resolved when issue #307 has been implemented.

**Comment by NickSchimek** [comment]
I know how to duplicate a similar `sqlalchemy.exc.OperationalError` error. Not the exact error you got, but similar `no such table` error. When backend tests are ran the test suite wipes the db. Since we use the same db for everything these issues can arise when trying to access resources after running a test. One way to duplicate is to run tests and then log into the flask shell and query a resource. 

The temporary fix for this issue is to recreate (create and seed) the DB.

I have a permanent fix for this which I'm pushing up shortly.
